packages/agoric-cli/src/start.js

```diff
@@ -306,7 +310,8 @@ export default async function startMain(progname, rawArgs, powers, opts) {
     for (const keyName of ['provision', 'delegate0']) {
       /* eslint-disable no-await-in-loop */
       let capret = showKey(keyName);
-      if (await capret[0]) {
+      const capretZero = await capret[0];
+      if (capretZero) {
```

Comment:
`capretZero` is certainly a stupid name. Someone who knows the purpose should suggest a better name.

Reply:
You can replace all occurrences of `capret` in this file with `statusOut`.  Then `statusOut[0]` is a promise for a process's `exitStatus`, and `statusOut[1]` is the (incrementally-updated) captured output of the process, which has a final value when `statusOut[0]` settles.

Reply:
Done